scripts/build.sh

```diff
@@ -19,4 +19,12 @@ export CGO_ENABLED=0
 
 # Build binary
 cd $GOPATH/src/github.com/raintank/metrictank
-go build -ldflags "-X main.GitHash=$GITVERSION" -o $BUILDDIR/metrictank
+
+OUTPUT=$BUILDDIR/metrictank
+
+if [ "$METRICTANK_BUILD" == "race" ]
```

Comment:
The advantage is that by using an env var I can also use other scripts like `make` and make them build with `-race`.